docs/REFERENCE.md

```diff
@@ -53,21 +76,40 @@ Interface common for both types of results: [result.mapAnyErr(callback)](#result
 
 The operation is the results version of [Promise.prototype.catch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch) function.
 
+**Examples:**
+
+```ts
+import { Ok } from 'type-safe-errors';
+
+const okResult = Ok.of(5);
+// No operation is performed, the original Ok result is returned
+const sameOkResult = okResult.mapAnyErr(err => 123); 
+
+```
+
 ---
 
 ### ok.unsafePromise()
 
-Map `Ok` to a fulfilled [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). 
-Interface common for both types of results: [result.unsafePromise()](#resultunsafePromise)
+Map an `Ok` result to a fulfilled [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).  
```

Comment:
Swap order of `ok.unsafePromise` and `ok.promise` 
Then it's more clear why it's unsafePromise